### PR TITLE
Updated Login Page for #230

### PR DESCRIPTION
**Pull request by uditguru19**

### CODE REVIEW

**Comment by velkymx** [comment]
@uditguru19 you have conflicts in the branch.

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -29,53 +27,111 @@
...
+
+                <button type="submit" class="btn btn-secondary btn-block mb-2 text-light text-uppercase submit-log">
+                    {{ __('Login') }}
+                </button>
+
+                <a class="btn btn-link text-primary pl-0 font-14" href="{{ route('password.request') }}">
+                    {{ __('Forgot Password?') }}
+                </a>
+                <div class="center mt-5">
+                    <p class="text-center mb-0 font-12">Dont have an account?</p>
```
Comment (velkymx) [inline resources/views/auth/login.blade.php]: @milaendo please make this a translation string.

File: resources/views/auth/login.blade.php [hunk 2]
```diff
@@ -29,53 +27,111 @@
...
+                </div>
+                <div class="text-center">
+                    <a class="btn-link text-primary pl-0 font-14" href="">
+                        Start your free trial now!
```
Comment (velkymx) [inline resources/views/auth/login.blade.php]: This one too

**Comment by milaendo** [comment]
screen shot 2018-06-13 at 1 12 52 pm

**Comment by milaendo** [comment]
screen shot 2018-06-13 at 1 21 37 pm

**Pull request merged**